dev-docs/RFCs/v6.0/gpu-screengrid-aggregation-rfc.md

```diff
@@ -44,6 +45,8 @@ Input:
 	* cellSizeChagned (Bool) : should be set to true when cellSize is changed.
 * countsBuffer: (Buffer, optional) : used to update aggregation data per grid, details in Output section.
 * maxCountBuffer: (Buffer, optional) : used to update total aggregation data, details in Output section.
+* projectPoints (Bool) : when true performs aggregation in screen space.
```

Comment:
Is this mainly for testing? Or related to screen grid layer?

Reply:
Yeah, this for screen space aggregation.